.github/workflows/package-unit-tests.yml

```
@@ -0,0 +1,40 @@
+name: Package Unit Tests
+
+env:
+  REALM_DISABLE_ANALYTICS: 1
+  MOCHA_REMOTE_TIMEOUT: 10000
+
+on:
+  pull_request:
+    paths:
+      # Source code
+      - "*"
+      - "lib/**"
+      - "src/**"
+      - "types/**"
+      - "react-native/**"
+      - "vendor/**"
+      # Integration tests
+      - "integration-tests/**"
+      # No need to run when updating documentation
+      - "!**.md"
+      # Run this on updates to the workflow
+      - ".github/workflows/integration-tests.yml"
```

These paths looks like they we're copied 1-to-1.

```suggestion
      # Source code
      - "packages/realm-network-transport/**"
      - "packages/realm-common/**"
      # No need to run when updating documentation
      - "!**.md"
      # Run this on updates to the workflow
      - ".github/workflows/package-unit-tests.yml"
```